### DISCUSSION TITLE
How do i access the gitpod of amrex ?

### DISCUSSION THREAD

**Discussion by pa-one55** [AMReX-Codes/amrex#4136] [Q&A]
I am not able to find any link.

**Comment by atmyers** [comment]
Hi,

We used Gitpod for an ECP All-hands tutorial in 2021. That should be available [here](https://github.com/atmyers/ecp-tutorials).

**Reply by pa-one55** [reply]
hey
how do i compile and run the codes ?
what commands are there ?
one time cmake .. is giving errors and other time its doing its work nicely.. what's the correct procedure ?
